app.ts

```diff
@@ -97,6 +97,10 @@ app.post("/register", (req: express.Request, res: express.Response) => {
     }
 })
 
+app.get("/api/getRaplaEvents/:course", (req: express.Request, res: express.Response) => {
```

Comment:
Das ist nicht Teil der API. Die Rückgabe des HTMLs muss über die GET Request im Calendar Router erfolgen. Hier müsste am besten ein neuere Parameter zur GET request hinzugefügt werden z.B. `timeline` was einen boolean darstellten könnte. Wenn dieser true wäre, dann soll das HTML für die Timeline zurückgegeben werden, wenn er false ist oder fehlt, dann das HTML für das Grid.

Was hältst du davon @maximiliani ?

Reply:
Ich stimme dir zu, dass dieser Endpoint nicht existiert. Dennoch bin ich der Meinung, dass man die API-Endpoints nicht mit Parametern  überladen sollte. Wir haben doch bereits den Start, End und Outputtype als Parameter. Das sollte reichen...

Reply:
Also soll dieser Endpoint bestehen bleiben? Das macht aber in Anbetracht der Consistency wenig Sinn. @maximiliani